redfish-core/lib/hypervisor_system.hpp

```diff
@@ -938,32 +938,13 @@ inline void handleHypervisorIPv6StaticPatch(
             return;
         }
 
-        bool errorInEntry = false;
-        if (address)
-        {
-            if (!ip_util::ipv4VerifyIpAndGetBitcount(*address))
-            {
-                messages::propertyValueFormatError(asyncResp->res, *address,
-                                                   pathString + "/Address");
-                errorInEntry = true;
-            }
-        }
-        else
+        if (!address)
         {
             messages::propertyMissing(asyncResp->res, pathString + "/Address");
             return;
         }
 
-        if (gateway)
-        {
-            if (!ip_util::ipv4VerifyIpAndGetBitcount(*gateway))
-            {
-                messages::propertyValueFormatError(asyncResp->res, *gateway,
-                                                   pathString + "/Gateway");
-                errorInEntry = true;
-            }
-        }
-        else
+        if (!gateway)
```

Comment:
Why do you have to patch with a gateway? 
Can't just do just the address, etc?

Reply:
This is downstream only code which was there in 1060 as well.
this was missed as part of rebase PRs.

Reply:
These changes were missed as part of 1110 rebase PR
https://github.com/ibm-openbmc/bmcweb/pull/938